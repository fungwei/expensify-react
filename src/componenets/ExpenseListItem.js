import React from 'react';

export default ({ description, amount, createdAt}) => (
  <div>
    <h3>Description: {description}</h3>
    <p>Amount: {amount}</p>
    <p>Created At: {createdAt}</p>
  </div>
);