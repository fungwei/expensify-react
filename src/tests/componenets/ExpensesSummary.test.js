import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../componenets/ExpensesSummary';

test('should render ExpensesSummary with expenses', ()=>{
  const wrapper = shallow(<ExpensesSummary expensesCount={2} expensesTotal={1235} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235} />);
  expect(wrapper).toMatchSnapshot();
});