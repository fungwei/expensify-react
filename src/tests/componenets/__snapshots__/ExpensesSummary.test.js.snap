// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should render ExpensesSummary with expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $12.35
  </h1>
</div>
`;
